### PR TITLE
Malnourished

### PR DESCRIPTION
**Pull request by Tetsukiba**
Adds **penalties** for being malnourished

- Lack of nutrition causes liver and stomach damage
- Lack of hydration causes liver and kidney damage
- These organs can begin to fail after 1-2 days in real time
- These organs will be destroyed after 2-3 days in real time

### CODE REVIEW

**Comment by genessee596** [comment]
Hey, Outreach trialmin here.

If you're gonna make hunger kill, please make it so hunger does not drain at all when someone is in a cryopod. At the moment, hunger drain is very slow (but not stopped entirely) in cryo, as hunger simply causes slow movement. If you're going to make it kill, this would be severely punishing for players who spend a long time in cryo for whatever reason.

TL:DR; Make players in cryo not get any hungrier until they leave cryo again.

**Comment by Tetsukiba** [comment]
> Hey, Outreach trialmin here.
> 
> If you're gonna make hunger kill, please make it so hunger does not drain at all when someone is in a cryopod. At the moment, hunger drain is very slow (but not stopped entirely) in cryo, as hunger simply causes slow movement. If you're going to make it kill, this would be severely punishing for players who spend a long time in cryo for whatever reason.
> 
> TL:DR; Make players in cryo not get any hungrier until they leave cryo again.

Implemented a change, hunger and thirst won't cause damage while player is in stasis, however hunger and thirst will still deplete.

**Comment by genessee596** [comment]
Good work

File: mods/persistence/modules/mob/malnourished.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	for(var/obj/item/organ/internal/kidneys/kidneys in src)
+		if((!hydration) && (!InStasis()))
+			kidneys.take_internal_damage(DEFAULT_HUNGER_FACTOR/1000)
+	. = ..()
+
+/mob/living/carbon/human/handle_regular_status_updates()
+	if((!nutrition || !hydration) && (!InStasis()))
+		add_chemical_effect(CE_TOXIN, 1)
+		adjustToxLoss(0.5001)
```
Comment (Andrew-Fall) [inline mods/persistence/modules/mob/malnourished.dm:18]: Why does this have the trailing .0001?

Comment (Tetsukiba) [reply mods/persistence/modules/mob/malnourished.dm:18]: an attempt to heal damage is constantly made, this .0001 allows slow damage to the liver. If the person is not in stasis they will slowly build up toxin and liver damage, which will take real days to do considerable damage. The heal attempt heals .5 per tick. the .0001 is the difference.

**Comment by Andrew-Fall** [comment]
I'll go ahead and create something similar on the latest branch, seeing as I never merged it before rebasing

**Pull request closed without merging**